core-common/src/main/java/org/mapstruct/MapperConfig.java

```diff
@@ -146,4 +146,17 @@ MappingInheritanceStrategy mappingInheritanceStrategy()
      * @return strategy how to do null checking
      */
     NullValueCheckStrategy nullValueCheckStrategy() default ON_IMPLICIT_CONVERSION;
+
+    /**
+     * If MapStruct could not find another mapping method or apply an automatic conversion it will try to generate a
+     * forged mapping method for the properties. If this property is set to {@code true} MapStruct will not try to
```

Comment:
perhaps we should make clear somehow that some times of mapping methods are still generated (iterable, map, stream, enum?) This only applies to bean properties, right? @gunnarmorling isn't that kind of confusing?